src/test/scala/org/broadinstitute/dsde/agora/server/dataaccess/acls/GcsAuthorizationSpec.scala

```diff
@@ -16,4 +16,13 @@ class GcsAuthorizationSpec extends FlatSpec {
     val entity = agoraBusiness.insert(testIntegrationEntity, "jcarey@broadinstitute.org")
     assert(agoraBusiness.findSingle(entity, Seq(entity.entityType.get), username) === entity)
   }
+
+  "Agora" should "check bucket write ACL when creating a new method in an existing workspace and fail if the user " +
+    "doesn't have access" in {
+    val thrown = intercept[AgoraAuthorizationException] {
+      val entity = agoraBusiness.insert(testIntegrationEntity, "ggrant@broadinstitute.org")
+      assert(agoraBusiness.findSingle(entity, Seq(entity.entityType.get), username) === entity)
```

Comment:
Do we need this line? The code should never reach here because an exception is thrown, right?